### PR TITLE
gh-125620: Skip check_resource_tracker_death on NetBSD due to long wait for SIGKILL process termination

### PR DESCRIPTION
**Pull request by furkanonder**
* Issue: gh-125620

### CODE REVIEW

**Comment by furkanonder** [comment]
In case of `time.sleep(60.0)` and `time.sleep(60.0 * 2)` the test still fails. For this reason `time.sleep(60.0 * 3)` has been set.

**Comment by furkanonder** [comment]
Additionally, while investigating the issue I [discovered](https://github.com/python/cpython/pull/121383/files#diff-42fd0a5b0b915b3c07ea405771fcf122210ff0ce5be1ad4b086d6016eec0330f) an unnecessary import. You can view the fix [here](https://github.com/python/cpython/pull/125624).

File: Lib/test/_test_multiprocessing.py [hunk 1]
```diff
@@ -5730,7 +5730,10 @@ def check_resource_tracker_death(self, signum, should_die):
         pid = _resource_tracker._pid
 
         os.kill(pid, signum)
-        time.sleep(1.0)  # give it time to die
+        if sys.platform.startswith("netbsd") and should_die:
+            time.sleep(60.0 * 3)  # give it more time to die on netbsd
+        else:
+            time.sleep(1.0)  # give it more time to die
```
Comment (rruuaanng) [inline Lib/test/_test_multiprocessing.py]: Can this comment be deleted? The above comments have explained the function of this statement.

Edit:
this `# give it more time to die`

File: Lib/test/_test_multiprocessing.py [hunk 2]
```diff
@@ -5730,7 +5730,10 @@ def check_resource_tracker_death(self, signum, should_die):
         pid = _resource_tracker._pid
 
         os.kill(pid, signum)
-        time.sleep(1.0)  # give it time to die
+        if sys.platform.startswith("netbsd") and should_die:
+            time.sleep(60.0 * 3)  # give it more time to die on netbsd
```
Comment (gpshead) [inline Lib/test/_test_multiprocessing.py]: do we know _why_ it takes so long?

Comment (vstinner) [reply Lib/test/_test_multiprocessing.py]: Would it be possible to use a test.support constant such as `support.SHORT_TIMEOUT`? It's 30 seconds by default.

3 minutes sounds way too long. I would suggest to mark the test as broken and skip it on NetBSD instead.

Comment (furkanonder) [reply Lib/test/_test_multiprocessing.py]: Unfortunately, a 30-second timeout is too short, and the test fails. I thought about using `support.LONG_TIMEOUT`, but which is over 3 minutes.

**Review by gpshead — APPROVED** [review]

File: Lib/test/_test_multiprocessing.py (hunk 2 above)
Comment (gpshead) [reply Lib/test/_test_multiprocessing.py]: thanks for changing it to a skip!

Comment (furkanonder) [reply Lib/test/_test_multiprocessing.py]: > thanks for changing it to a skip!

You are welcome!

**Pull request merged**